burst.sh

```diff
@@ -419,10 +419,6 @@ if [[ $# -gt 0 ]] ; then
             ;;
     esac
 else
-    ARCH=`uname -m`
-    if [[ $ARCH = "armv7l" ]]; then
-        export LD_LIBRARY_PATH=./lib/armv7l
-    fi
```

Comment:
not needed?

Reply:
The variable is not referenced anywhere else, so I think not.